lta/rest_server.py

```diff
@@ -163,6 +163,45 @@ class TransferRequestActionsPopHandler(BaseLTAHandler):
                     break
         self.write({'results': ret})
 
+class StatusHandler(BaseLTAHandler):
+    @lta_auth(roles=['admin', 'user', 'system'])
+    async def get(self) -> None:
+        """Get the overall status of the system"""
+        ret = {}
+        health = 'OK'
+        old_data = (datetime.utcnow() - timedelta(seconds=60*5)).isoformat()
```

Comment:
This seems like another one you'd want a configuration knob for.
Maybe `LTA_STALE_STATUS_SECONDS` ?

Reply:
Yeah, I thought about it, then decided it was not straightforward to implement (challenging to pass that value through to this location) and pushed it down the road.  It'll probably happen in a later refactoring when we impose stricter limits on monitoring quantities.